src/pages/extending-chat-widget/chat-widget-moments/index.mdx

```diff
@@ -184,6 +185,24 @@ This method has no parameters.
 momentsSDK.close();
 ```
 
+### setIsFragile
+
+Set `isFragile` value. It can be provided in initalization and changed later with this function.
```

Comment:
```suggestion
Set `isFragile` value. It can be provided as part of initialization data and changed later with this function.
```
apart from wording we could additionally reference a link to `init` payload